package.json

```diff
@@ -33,6 +33,7 @@
     "lodash": "~2.4.1",
     "mkdirp": "^0.5.0",
     "moment": "~2.7.0",
+    "otr": "^0.2.15",
```

Comment:
I noticed this package is in [license MPL 2.0](https://www.mozilla.org/en-US/MPL/2.0/). I don't know this one, do you think it could conflict with the rest of Shout?